.changeset/hot-maps-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ebay/ebayui-core": patch
```

Comment:
Should be major

Reply:
I think this is actually because `14.0.0` does not match with `master`, I didn't add a changeset